tests/test_amin_amax.py

```diff
@@ -79,14 +64,14 @@ def _get_min_max_input(type, shape):
 @pytest.mark.parametrize(
     "shape", [(4,), (2, 3), (4, 5, 6)], ids=["(4,)", "(2,3)", "(4,5,6)"]
 )
-def test_amax(type, shape):
+def test_amax_diff_shape(type, shape):
     a = _get_min_max_input(type, shape)
 
     ia = dpnp.array(a)
```

Comment:
use `get_all_dtypes()` for parametrizing `type`, otherwise dpnp will test twice with "float32" on Iris Xe